dotenv/mod.ts

```diff
@@ -57,7 +57,7 @@ export function parse(
 
     env[key] = notInterpolated
       ? notInterpolated
-      : interpolated
+      : interpolated || interpolated === ""
```

Comment:
Could this become `typeof interpolated  === "string"`?

Reply:
That's a lot more explicit, let's do that.